mtags/src/main/scala-2/scala/meta/internal/pc/completions/FilenameCompletions.scala

```diff
@@ -7,11 +7,8 @@ import scala.collection.immutable.Nil
 import scala.util.control.NonFatal
 
 import scala.meta.internal.pc.CompletionFuzzy
-import scala.meta.internal.pc.CompletionFuzzy
```

Comment:
Random side note, the next release of OrganizeImports will also remove duplicated ones like this 🥳

Reply:
I think that was actually removed by that rule :sweat_smile:

Reply:
Or maybe it was due to warnings I saw?

Reply:
O really? Haha, nevermind then. I thought you did it manually.